### PR TITLE
CHART website 2.0 

### PR DESCRIPTION
**Pull request by dannyjacobs**
revamped to use markdown.
- tried out a couple different templates, currently using a lightly modified one from zendesk
- ported all construction tutorials that were still relevant
- brought in latest publications page
- have tested deployment on Danny's machine and Libby's machine

Known issues:
-  CHART  appears halfway down the screen instead of upper left corner. 
- @lmberkhout wants to add more to analysis section

Heres what it looks like now.
image

Ready for review and comment!

### CODE REVIEW

File: wiki/resources.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Resources
+===
+### Computer help!
+We will make moderate use of the linux command line terminal. Through the terminal we can do all the usual things we do
+n a computer like inspect the contents of directories, manage files, and run programs. The terminal itself is a program
+called Bash (which is a portmanteu of Bourne Again SHell). If this is your first time in the shell, check out our
+[linux shell🐢 help page](shell_help).  
+
+We were all n00bs once! See a bit slang you don't recognize? Consult our modest [compendium of computer
+vocabulary](assets/vocab).  Or just, you
```
Comment (adampbeardsley) [inline wiki/resources.md:13]: `/assets/vocab not found.`

**Review by adampbeardsley — COMMENTED** [review]
Some general comments/todos:

- [x] We need to untrack a lot of the files that the server creates, and add things to .gitignore. After testing the other day, I just tried pulling and it was a pain with merge conflicts.
- [ ] Can we get the favicon working again?
- [ ] Should the "old" directory be removed?
- [ ] logo on sidebar is weirdly placed and behind the gray bar, making it hard to see.
- [ ] We can drop the footer acknowledgement in favor of a list on the About page.
- [ ] I think we should re-add some of the GUI walkthrough to the website. The observing document lists the steps, but I think it's nice to have the step-by-step with screenshots right on the site _somewhere_
- [ ] The GNU radio flowgraphs aren't a part of the main path anymore, but I liked having them as a part of the "hackable" aspect - it allows someone who's more interested in the signal processing to dig a little deeper there. Can we add a section for non-main-quest tutorials?
- [ ] I know this is the public-facing website, but we should point to the actual repo somewhere in case anyone wants to dig deeper and/or contribute.

**Comment by adampbeardsley** [comment]
oh, and I checked it on an ios simulator and it seemed to look good. fingers crossed it will actually work on various mobile devices.

File: wiki/about.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+Meet the Team
+---------------------------
+
+![Ahlea](assets/ahlea.jpg){:height="50%" width="50%"}
+
+Ahlea Wright
+------------
+
```
[the remark this answers is not in the document]
Comment (lmberkhout) [reply wiki/about.md:19]: I am fine with either. The side-by-side is probably better.

**Review by adampbeardsley — APPROVED** [review]

**Pull request merged**